### ISSUE TITLE
Post loop builder Filter bar filtering bug

### ISSUE THREAD

**Issue by ElinaMacichina** [soflyy/breakdance-bugs#1011]
1. Create post loop builder with filter bar in it
2. Click on created filter bar options (on my example there are some post categories)
3. You will see that some blocks that need to be shown often goes randomly and stuck somewhere. 

I don't know if it bug or feature, but when filter bar is on, blocks are shown in vertical masonry grid and nowhere you can select this or change to normal grid.

![image](https://github.com/soflyy/breakdance-bugs/assets/92719267/ec2c9735-10a8-44ca-bf30-59680481d1a6)

You should also include the following information:
 - Breakdance version: 1.6.1
 - Firefox, Chrome, Windows 11

**Comment by adkoen** [comment]
YES, I came here to file the same issue. Using the latest version of Breakdance, whenever you activate the filter bar, the grid view for the posts is not working as expected. It turns into some sort of Masonry instead of just a regular grid. I tried to override it with CSS, but there is a lot of Javascript going on, settings widths an heights dynamically in the HTML. I can't fix it.

Screenshot of the grid layout with the Filter bar active. Notice the 4th post. It should be outlined on the left, just like a regular grid.
<img width="1350" alt="SCR-20240306-ntxh" src="https://github.com/soflyy/breakdance-bugs/assets/7188497/1cf9c7fe-d713-41df-be0f-97f5cde01cf3">
